enclave_build/src/lib.rs

```diff
@@ -23,8 +23,8 @@ pub const DEFAULT_TAG: &str = "1.0";
 
 #[derive(Debug, Error)]
 pub enum EnclaveBuildError {
-    #[error("Docker error: `{0:?}`")]
-    DockerError(DockerError),
+    #[error("Docker error: `{0}`")]
+    DockerError(String),
```

Comment:
Why not use `thiserror` for DockerError as well and transform it to a formatted string instead of having strings all over the place?

Reply:
That would mean keeping the DockerError type in `docker.rs` and passing strings at that level. Wasn't the idea of the refactoring to get rid of specific error type definitions? I can define multiple docker errors here.

Reply:
Sure, but this commit basically transforms many of the structured errors into unstructured ones via String.

Reply:
I gave an example above.

Reply:
Right, I'll make it more structured.